File: app/src/app/core/models/course.ts
/*
 * Este fichero forma parte de UBUMonitorWeb.
 *
 * @author Marcelo Verteramo Pérsico
 */

/** Modelo de curso devuelto por el backend. */
export type Course = {
  id: number;
  name: string;
  picture: string;
  category: string;
  isStarred: boolean;
  sinceTs: number | null;
  untilTs: number | null;
};

File: api/src/main/kotlin/es/ubu/lsi/ubumonitorweb/data/api/Course.kt
/*
 * Este fichero forma parte de UBUMonitorWeb.
 *
 * @author Marcelo Verteramo Pérsico
 */

package es.ubu.lsi.ubumonitorweb.data.api

data class Course(
  val id: Int,
  val name: String,
  val picture: String?,
  val category: String,
  val isStarred: Boolean,
  val sinceTs: Long?,
  val untilTs: Long?,
)
